src/features/applets-refresh/ui/AppletsRefresh.tsx

```diff
@@ -15,6 +15,7 @@ type Props = Omit<RefreshControlProps, 'refreshing' | 'onRefresh'>;
 
 const AppletsRefresh: FC<Props> = props => {
   const queryClient = useQueryClient();
+  const [appletsRefreshing, setAppletsRefreshing] = useState(false);
```

Comment:
I'll have to check this fix

Reply:
I've checked on ios simulator. The spenner is not shown now, when pull-to-refresh.